src/app/services/ServicesRecapPage.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+
+import ListItem from './ListItem';
+import Header from '../../components/header';
+import SectionHeader from '../../components/sectionHeader';
+
+class ServicesRecapPage extends Component {
+  getServiceItems = (category) => {
+    const { services } = this.props;
+
+    return Object.keys(services[category]).map((id) => {
+      const service = services[category][id];
+
+      return service.selected ? (
+        <ListItem key={id} title={service.name} progress={service.progress || 0} />
+      ) : (
+        undefined
```

Comment:
It at least used to be that React would only agree to render null, but not undefined.
I guess that might have changed though.

Reply:
Code was removed with changes

Reply:
I thought I still saw it somewhere else (maybe one of the other PRs?), though either way it makes little difference.